test/stack/10-hdfs.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: simple-zk
+spec:
+  image:
+    productVersion: 3.8.3
+  servers:
+    roleGroups:
+      default:
+        replicas: 1
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperZnode
+metadata:
+  name: simple-hdfs-znode
+spec:
+  clusterRef:
+    name: simple-zk
+---
+apiVersion: hdfs.stackable.tech/v1alpha1
+kind: HdfsCluster
+metadata:
+  name: simple-hdfs
+spec:
+  image:
+    productVersion: 3.3.6
+    custom: hdfs
+  clusterConfig:
+    dfsReplication: 1
+    zookeeperConfigMapName: simple-hdfs-znode
+    authentication:
+      tlsSecretClass: tls
+      kerberos:
+        secretClass: kerberos-default
+  nameNodes:
+    roleGroups:
+      default:
+        envOverrides:
+          HADOOP_CLASSPATH: "/stackable/hadoop/share/hadoop/tools/lib/hdfs-group-mapper-0.1.0-SNAPSHOT.jar"
+        configOverrides:
+          core-site.xml:
+            # the mapper is only handled on the namenode so no need to apply this config to all roles
+            hadoop.security.group.mapping: "tech.stackable.hadoop.StackableGroupMapper"
+            hadoop.security.group.mapping.opa.url: "http://test-opa.default.svc.cluster.local:8081/v1/data/hdfs-groups"
```

Comment:
I'm sorry, I missed the `hadoop.security.group.mapping.opa.list.name` setting below.
I would leave it to you if you want to revert 0bd35a2be3e9c69f8338a3faecd6e0a83c381c9c

Reply:
no problem, I'll leave it as it is.